### ISSUE TITLE
 Login Page UX and Accessibility

### ISSUE THREAD

**Issue by ShraddhaBhujbal11** [coderashhar/PG-genie#76]
The current Login page works but lacks some essential UX and accessibility features. This enhancement will:

->Show a visible error message when username or password is left empty

->Add proper labels for form inputs to improve accessibility

->Replace hardcoded/incorrect Tailwind padding class (p-15) with a valid one

->Use React Router <Link> instead of <a> for navigation to avoid full page reloads

->Clean up repeated Tailwind styles for maintainability
 

<img width="1357" height="823" alt="Image" src="https://github.com/user-attachments/assets/6fc846e9-ffdf-415a-b011-127122f047cb" />

Need of these points:-
These improvements will make the login experience smoother, more user-friendly, and accessible to all users, while also improving the code’s maintainability.

**Comment by ShraddhaBhujbal11** [comment]
GSSoC 2025

**Comment by Gunjan10-droid** [comment]
@coderashhar could you assign me this issue?

**Comment by neha-reddy05** [comment]
I would like to work on this. I am GSSOC contributor

**Comment by ShraddhaBhujbal11** [comment]
Please assign this issue to me...as this feature will enhance the project 
GSSoC 2025
